pkg/server/commands/read_changes.go

```diff
@@ -63,8 +65,29 @@ func (q *ReadChangesQuery) Execute(ctx context.Context, req *openfgav1.ReadChang
 	if err != nil {
 		return nil, serverErrors.InvalidContinuationToken
 	}
+	token := string(decodedContToken)
+
+	var startTime time.Time
+	if req.GetStartTime() != nil {
+		startTime = req.GetStartTime().AsTime()
+	}
+	if token == "" && !startTime.IsZero() {
+		tokenUlid, ulidErr := ulid.New(ulid.Timestamp(startTime), nil)
```

Comment:
Anything we should note/comment about the importance of `nil` entropy?

Reply:
At this point it only impacts filtering records within the same millisecond. Starting from zero makes sense for filtering by time.